autoload/flood/check.vim

```diff
@@ -94,10 +129,20 @@ function! flood#check#run(...) abort
   endif
 
   let mode = a:0 > 0 ? a:1 : 'r'
-  let cmd = printf('%s --json', flood#flowbin())
-  let s:job = job_start(cmd, {
-        \ 'callback': {c, m -> s:check_callback(c, m, mode)},
-        \ })
+  let cmd = [flood#flowbin(), '--json']
```

Comment:
Bit a tricky.
Move this into `if has('nvim')` block.
And Vim's block, use `let cmd = printf('%s --json', flood#flowbin())`